backend/.env.example

```
@@ -1,5 +1,4 @@
-PORT=3000
-SUPABASE_URL=https://your-project.supabase.co
-SUPABASE_SERVICE_KEY=your-service-role-key-here
-N8N_WEBHOOK_URL=https://your-n8n-instance.app.n8n.cloud/webhook/kaamchalu-events
-AI_SERVICE_URL=http://localhost:4000
+NEXT_PUBLIC_SUPABASE_URL=https://biytwqtyjfkaudzblbyy.supabase.co
+NEXT_PUBLIC_SUPABASE_PUBLISHABLE_DEFAULT_KEY=sb_publishable_rz5SmJJngZoBmSVqmxUEdw_Gw-Zaj1k
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_PUBLISHABLE_DEFAULT_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)